Common/src/CConfig.cpp

```diff
@@ -5165,12 +5177,12 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
   else {monoatomic = false;}
 
   /*--- Set number of Turbulence Variables. ---*/
-  switch (TurbModelFamily(Kind_Turb_Model)) {
-    case TURB_FAMILY::NONE:
+  switch (Kind_Turb_Model) {
```

Comment:
in the proposed change, TURB_FAMILY would be unnecessary since we will only have NONE, SA or SST turb models.